docs/en/revocation_endpoint.rst

```diff
@@ -1,16 +1,31 @@
 .. include:: ../common/common_definitions.rst
 
+.. _Revocation_Endpoint:
+
 Revocation Endpoint
 -------------------
 
 An RP MAY request the revocation of an Access Token or a Refresh Token issued by an OP.
 
-When the user logs out, the RP MUST revoke the Access Token in its possession.
+The OP MUST revoke the token specified in the request.
 
-.. note::
-  The Access Token revocation implies revoking all the Refresh Tokens linked to it.
+.. admonition:: |spid-icon|
+
+  When the user logs out or when his/her session with the RP expires (according to the policies decided by the RP) the RP MUST revoke the Access Token in its possession, if it is not expired yet.
+
+  .. note::
+    The Access Token revocation implies revoking all the Refresh Tokens linked to it, if not expired yet.
+
+  The OP will have to revoke the token specified in the request and will have to end the Single Sign-On session, if it is still active. Any other active tokens for the user must be kept valid.
```

Comment:
non capisco a cosa si riferisce "Any other active tokens for the user must be kept valid."

se abbiamo detto che la revoca di un AT comporta la revoca di tutti i refresh collegati, quali altri token dovrebbero risultare attivi?

Reply:
Mi sembra che qui il discorso fosse relativo alla revoca del Refresh Token, che non deve incidere su Access Token e ID token,  alla revoca dell'ID Token che non deve incidere sull'Access Token e viceversa.

Reply:
lasciamo as-is, in previsione dei futuri lavori su SSO ed estensione del refresh lato AgID.